frontend/src/pages/ShootItemEditor.vue

```diff
@@ -0,0 +1,485 @@
+          this.snackbarColor = 'success'
+          this.snackbarText = `Cluster specification has been successfully updated`
+          this.snackbar = true
+        } catch (err) {
+          this.alert = true
+          this.alertType = 'error'
+          this.alertMessage = err.message
+        }
+      },
+      undo () {
+        if (this.$instance) {
+          this.$instance.execCommand('undo')
+        }
+      },
+      redo () {
+        if (this.$instance) {
+          this.$instance.execCommand('redo')
+        }
+      },
+      reload () {
+        this.update(this.value)
+      },
+      downloadContent () {
+        try {
+          download(this.getContent(), this.getQualifiedName(), 'text/yaml')
+          this.snackbarColor = undefined
+          this.snackbarText = 'Content has been downloaded'
+          this.snackbar = true
+        } catch (err) {
+          this.snackbarColor = 'error'
+          this.snackbarText = 'Download content failed'
+          this.snackbar = true
+        }
+      },
+      refresh () {
+        this.$nextTick(() => this.refreshInstance())
+      },
+      refreshInstance () {
+        if (this.$instance) {
+          this.$instance.refresh()
+        }
+      },
+      createInstance (element) {
+        const vm = this
+        const extraKeys = {
+          'Ctrl-S' (instance) {
+            vm.save()
+          },
+          'Cmd-S' (instance) {
+            vm.save()
+          },
+          'Tab': (instance) => {
+            if (instance.somethingSelected()) {
+              instance.indentSelection('add')
+            } else {
+              instance.execCommand('insertSoftTab')
+            }
+          },
+          'Shift-Tab': (instance) => {
+            instance.indentSelection('subtract')
+          }
+        }
+        const options = {
+          mode: 'text/x-yaml',
+          autofocus: true,
+          indentUnit: 2,
+          tabSize: 2,
+          indentWithTabs: false,
+          smartIndent: true,
+          scrollbarStyle: 'native',
+          lineNumbers: true,
+          lineWrapping: true,
+          extraKeys
+        }
+        this.$instance = CodeMirror(element, options)
+        this.$instance.setSize('100%', '100%')
+        const onChange = ({doc}) => {
+          this.untouched = false
+          this.clean = doc.isClean(this.generation)
+          this.historySize = doc.historySize()
+          this.alert = false
+          this.alertMessage = ''
+        }
+        this.$instance.on('change', onChange)
+      },
+      destroyInstance () {
+        if (this.$instance) {
+          const element = this.$instance.doc.cm.getWrapperElement()
+          if (element && element.remove) {
+            element.remove()
+          }
+        }
+        this.$instance = undefined
+      },
+      setOption (key, value) {
+        if (this.$instance) {
+          this.$instance.setOption(key, value)
+        }
+      },
+      getOption (key) {
```

Comment:
remove (currently) unused setOption and getOption

Reply:
:heavy_check_mark: